runner.py

```diff
@@ -17,10 +17,12 @@ def runner(input_queue, output_queues, queue_selector_path, print_summary,
   import torch
   from queue import Empty, Full
   from tqdm import tqdm
-  from rnb_logging import logname, TimeCardSummary, TimeCard
+  from rnb_logging import logname, TimeCardSummary, TimeCard, TimeCardList
   from control import TerminationFlag, Signal
   from utils.class_utils import load_class
 
+  torch.backends.cudnn.benchmark = True
```

Comment:
This flag is known to affect performance (https://discuss.pytorch.org/t/what-is-the-differenc-between-cudnn-deterministic-and-cudnn-benchmark/38054/2). Although we empirically know that enabling this flag can speed up inference, I think it'd be better to add comments and investigate the effect of this flag as a separate issue (probably an intern task? :)

I'd like to file an issue and add TODO comment if you don't mind.

Reply:
Yes, that seems a good idea. Please do.